packages/phones/src/phone-field/PhoneFieldMenu.tsx

```diff
@@ -20,17 +20,18 @@ const useStyles = makeStyles(
 export interface PhoneFieldMenuProps extends Pick<MenuProps, 'anchorEl'> {
   onClose: () => void;
   value: CountryISO;
+  disabled?: boolean;
```

Comment:
We should not put disabled here. Instead lets move this logic to PhoneFieldStartAdornment

Reply:
updated